operations/iuf/workflows/management_rollout.md

```diff
@@ -21,6 +21,13 @@ Once this step has completed:
 
 - Host firmware has been updated on management nodes
 
+Note: In CSM 1.6, all the worker nodes are configured as iSCSI SBPS targets. Starting CSM 1.7.0, selective worker node
```

Comment:
This doesn't make sense being under the FAS step of this procedure. It should probably go at the top of step 2, under a new subsection.